style.css

```diff
@@ -41,8 +42,14 @@ h1 > a:hover {
   color: #ddd;
 }
 
-p, ul {
-  font-weight: 300;
+h1 > a:focus {
```

Comment:
Instead of repeating each declaration block, could you comma-separate the selectors?

ex:
```css
a:hover,
a:focus {
  ✨style properties here✨
}
```

Reply:
Ugh, duh. I learned SASS very early on so I forget what functionalities vanilla CSS has... But yes!